tests/behat/features/prof/03-report/09-deputy-discharged.feature

```diff
@@ -0,0 +1,202 @@
+Feature: Prof deputy is discharged
+
+
+  Scenario: Case manager discharges professional deputy from client
+    Given I am logged in to admin as "casemanager@publicguardian.gov.uk" with password "Abcd1234"
+    And I discharge the deputies from case "01000010"
```

Comment:
Can we add some earlier test steps to ensure that the discharged regions aren't shown _before_ the deputy is discharged?

Also, this test should propbably use more Whens and Thens.

Reply:
Amended tests, look much better now. It also highlighted the discrepancy when two clients of same case number appear in the list. Ive amended behat regions these with "discharged-client-XXXX" to allow tests to click on the appropriate region.